### PR TITLE
[BAU] fixes from design testing

### PR DESCRIPTION
**Pull request by cyrusdobbs**
https://dluhcdigital.atlassian.net/wiki/spaces/FS/pages/50561200/Design+Team+testing

### Change description
- fix sign out button content
- fixes upload page width
- fixes content around re-uploading on error page

- [ ] Unit tests and other appropriate tests added or updated
- [X] README and other documentation has been updated / added (if needed)
- [X] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: app/templates/main/upload.html [hunk 1]
```diff
@@ -15,110 +15,114 @@
 {% block content %}
 
     <main id="main-content" role="main">
+                {% if not validation_errors %}
```
Comment (Connor-OS) [inline app/templates/main/upload.html:18]: Is this section double indented?

File: app/templates/main/upload.html [hunk 2]
```diff
@@ -15,110 +15,114 @@
...
 
-                {{ govukButton({"element": "button",
-                "text": "Upload and check for errors",
-                "attributes": {
-                    "aria-controls": "example-id",
-                    "data-tracking-dimension": 123} }) }}
-            </form>
-            {{ helpLinksDropdown() }}
-        </div>
+                <div class="upload-data-container govuk-!-margin-top-5">
```
Comment (Connor-OS) [inline app/templates/main/upload.html:95]: Same her I think

Comment (cyrusdobbs) [reply app/templates/main/upload.html:95]: this is correct - if you check the closing tags you can see it lines up

File: app/templates/main/upload.html (hunk 1 above)
Comment (cyrusdobbs) [reply app/templates/main/upload.html:18]: fixed

**Review by Connor-OS — APPROVED** [review]
LGTM

**Pull request merged**